src/api/Flooq.Api/Controllers/ContactController.cs

```diff
@@ -0,0 +1,80 @@
+#nullable disable
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Flooq.Api.Models;
+using Flooq.Api.Services;
+
+namespace Flooq.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ContactController : ControllerBase
+    {
+        private readonly IContactService _contactService;
+
+        public ContactController(IContactService contactService)
+        {
+          _contactService = contactService;
+        }
+
+        // GET: api/Contact
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<Contact>>> GetContacts()
+        {
+          return await _contactService.GetContacts();
+        }
+
+        // GET: api/Contact/5
+        [HttpGet("{id}")]
+        public async Task<ActionResult<Contact>> GetContact(string id)
+        {
+          var contact = await _contactService.GetContact(id);
+          return contact.Value == null ? NotFound() : contact;
+        }
```

Comment:
I don't think that is needed because for now we only need a list of contacts. You can remove this endpoint.